app/views/users/_list.haml

```diff
@@ -0,0 +1,11 @@
+%tbody
```

Comment:
Do our `/_list.haml`s usually not contain the header? (If they don't, interesting! I thought they did.)

Reply:
Btw, can we pass in `users` as a local variable instead of using `@users`? Would be nice to start relying on strict locals more for our partials!

Reply:
Some partials do, some don't

Reply:
Can we include it here if possible? I think it makes sense to consistently include it, where possible

Reply:
Awkward to combine because of differing header formats (search bar vs other things); nm!